libs/ui-watt/src/lib/components/input/date-range-input/watt-date-range-input.component.scss

```diff
@@ -0,0 +1,63 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+@use '@energinet-datahub/watt/utils' as watt;
```

Comment:
question: should we prefer relative imports when we're referencing files within the same library (i.e. Watt)?

Reply:
I was actually thinking of the same argument for using relative imports :D - it makes it easier to rename the namespace, or allowing the library to be extracted to its own package. So one place less to update. But I guess both of those options are very unlikely to happen. So let's leave it as is.